containers/dockerfile_gpuhoomd3conda

```diff
@@ -0,0 +1,80 @@
+FROM nvidia/cuda:11.6-devel 
+
+# Install system dependencies
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+		libeigen3-dev \
```

Comment:
not sure if it'll matter, but it looks like there's an extra space here

Reply:
Yeah you're right, on the `libeigen3-dev` line? When I open it up in vim it looks aligned with the line above. I wonder if it's just because of the font used on github or something?

Reply:
It might be a tabs vs spaces issue? https://stackoverflow.com/questions/1675688/make-vim-show-all-white-spaces-as-a-character